### PR TITLE
Remove search-api's neo4j-to-es-attributes.json and replace with indexed: indicators on attributes in entity-api's provenance_schema.yaml

### PR DESCRIPTION
**Pull request by kburke**
Initial changes for search-api/issues/753. Remove search-api's neo4j-to-es-attributes.json and replace with indexed: indicators on attributes in entity-api's provenance_schema.yaml.

### CODE REVIEW

File: src/schema/provenance_schema.yaml [hunk 1]
```diff
@@ -453,6 +503,7 @@ ENTITIES:
         generated: true
         transient: true
         immutable: true
+        indexed: true
```
Comment (yuanzhou) [inline src/schema/provenance_schema.yaml:506]: Do not index `previous_revision_uuids`

File: src/schema/provenance_schema.yaml [hunk 2]
```diff
@@ -469,6 +521,7 @@ ENTITIES:
...
```
Comment (yuanzhou) [inline src/schema/provenance_schema.yaml:524]: Do not index `next_revision_uuids`

File: src/schema/provenance_schema.yaml [hunk 3]
```diff
@@ -88,6 +94,7 @@ ACTIVITIES:
       <<: *shared_properties
       creation_action:
         type: string
+        indexed: true
```
Comment (yuanzhou) [inline src/schema/provenance_schema.yaml:97]: Out index procedure doesn't handle Activity nodes. This `creation_action` from Dataset and Publication does get indexed.

**Review by yuanzhou — CHANGES_REQUESTED** [review]

File: src/schema/provenance_schema.yaml (hunk 3 above)
Comment (kburke) [reply src/schema/provenance_schema.yaml]: Pushed these corrections.  I assume comment above should say, "... `creation_action` ... does **_NOT_** get indexed."

**Review by yuanzhou — APPROVED** [review]

**Pull request merged**